pcdsdevices/sequencer.py

```diff
@@ -189,3 +199,131 @@ def stop(self):
         """Stop the EventSequencer"""
         logger.debug("Stopping the EventSequencer")
         self.play_control.put(0)
+
+
+class SequenceLine(Device):
+    """Sub-class for event sequencer line."""
+
+    ec = FCpt(EpicsSignal, '{self.prefix}:EC_{self._ID}:{self._line}')
+    bd = FCpt(EpicsSignal, '{self.prefix}:BD_{self._ID}:{self._line}')
+    fd = FCpt(EpicsSignal, '{self.prefix}:FD_{self._ID}:{self._line}')
+    bc = FCpt(EpicsSignal, '{self.prefix}:BC_{self._ID}:{self._line}')
+
+    def __init__(self, prefix, hutch_id=None, line=None, **kwargs):
+
+        self._ID = hutch_id
+        self._line = line
+
+        super().__init__(prefix, **kwargs)
+
+    def read(self):
+        """Read this line of the event sequence. 
+
+        Parameters
+        ----------
+        None.
+
+        Examples
+        --------
+        SequenceLine.read()
+
+        """
+        event_code = self.ec.get()
+        beam_delta = self.bd.get()
+        fiducial_delta = self.fd.get()
+        burst_count = self.bc.get()
+        
+        line = [event_code, beam_delta, fiducial_delta, burst_count]
+
+        return line
+
+    def write(self, line):
+        """Write to this line of the event sequence. 
+
+        Parameters
+        ----------
+        line: list
+            Four item list containing the desired values for event code,
+            beam delta, fiducial delta, and burst count, respectively, e.g.
+            [<event code>, <beam delta>, <fiducial delta>, <burst count>].
+
+        Examples
+        --------
+        SequenceLine.write([140, 12, 0, 0])
+
+        """
+
+        if len(line) != 4:
+            raise ValueError("The sequence line must be a 4 item list!")
```

Comment:
One more thought: it'd be cool to be able to use a 5th slot for the comment/DESC field!